docs/source/examples/checks/distribution/whole_dataset_drift.ipynb

```diff
@@ -5,15 +5,66 @@
    "id": "104dbd2d",
```

Comment:
<p>Maybe replace</p><blockquote>A condition is deepchecks' way to enforce that results are OK, and we don't have a problem in our data or model!</blockquote><p>with</p><blockquote>A condition is deepchecks' way to validate model and data quality, and let you know if anything goes wrong.</blockquote><p>Or something like that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>